### DISCUSSION TITLE
Help, can this be fixed?

### DISCUSSION THREAD

**Discussion by stepsteph** [thatskymod/Sky-CotL-Scripts#51] [General]
This resent, new season, I got a bad problem while using a lib. (Libdukun.so )
I tried to complete these season of revival kinda fast, but not knowing what could happen next, I used this lib in the 'shop spirit' section and did all the season quests and after I did the whole thing, Im unable to see some things without the hack...

![Screenshot_20231023_004921_Sky](https://github.com/thatskymod/Sky-CotL-Scripts/assets/146548916/9f635030-5821-4b45-a6bd-d3ee1fd21073)
![Screenshot_20231023_005142_Canvas](https://github.com/thatskymod/Sky-CotL-Scripts/assets/146548916/4400f969-7e86-4613-be69-9c28f8bc3986)
![Screenshot_20231023_004953_Sky](https://github.com/thatskymod/Sky-CotL-Scripts/assets/146548916/1a156c0c-9c47-4304-8e5a-2898dc84211f)
![Screenshot_20231023_005210_Canvas](https://github.com/thatskymod/Sky-CotL-Scripts/assets/146548916/0a441276-559c-41ea-a695-d38cc9a3f2a5)

In other words,..... I'm stuck with the hack and I dunno where to report this 'shop spirits' thing. So my expectations was see it completed without the hack. I don't want to see anything empty.

Sad that I can't rewind or replay these to fix it...

**Comment by thatskymod** [comment]
Hi @stepsteph!

Long time modder here, so I thought I'd share my insights on this situation.

It seems you're having trouble seeing seasonal content from earlier stages of the active season.

First, once you obtain a collectible or complete a quest in Sky, these actions cannot be undone from the consumer end.

This means: Without the proper tools and knowledge, rolling back the season to an earlier stage is nearly impossible.

Sky has an API endpoint for deleting unlocks, which can be found by delving into the game's code. While this endpoint might be used internally, existing mods and techniques don't provide an interface to trigger this functionality.

You could try disabling "Closet & Emotes" in libsmite, as it's known to interfere with or inadvertently advance the season. Also, try visiting the area via the original Sky app to rule out any potential account issues.

I'm sorry to hear about this situation, and I hope a solution for issues like yours can be found soon.

**Reply by stepsteph** [reply]
Thanks for reaching out to me. I was able to fixed the problem following
your instructions; I opened the original sky and put relieve memory in the
second seasonal quest, it let me do the quest... "again " and let me see it like
it needs to.

![Screenshot_20231027_005711_Sky](https://github.com/thatskymod/Sky-CotL-Scripts/assets/146548916/15eb76da-5702-41a6-a8a4-10d36b3b3912)
